eng/Version.Details.xml

```diff
@@ -89,13 +89,13 @@
       <Uri>https://github.com/nuget/nuget.client</Uri>
       <Sha>19756345139c45de23bd196e9b4be01d48e84fdd</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="4.12.0-3.24510.2">
+    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="4.13.0-1.24517.9">
```

Comment:
@dotnet/kitten can you go over our darc subscriptions? we should have been on Roslyn 4.13 already.

Reply:
the subscription to roslyn vs17.13 wasn't created
fixed: https://github.com/dotnet/msbuild/pull/10852/files

Reply:
feel free to pull the changes from https://github.com/dotnet/msbuild/pull/10852 

I will merge it later today.